src/plugins/intel_cpu/src/graph.cpp

```diff
@@ -241,7 +241,7 @@ void Graph::Replicate(const std::shared_ptr<const ov::Model>& model,
         const auto& inputNode = input.second;
         const auto precToSet = inputNode->getOriginalOutputPrecisionAtPort(0);
         const auto childEdges = inputNode->getChildEdgesAtPort(0);
-        for (size_t i = 0; i < childEdges.size(); i++) {
+        for (size_t i = 0; i < childEdges.size(); i++) {  // NOLINT(modernize-loop-convert)
```

Comment:
please double check

Reply:
Yes, it seems to be obsolete also. Removed